docs/home.md

```diff
@@ -35,3 +35,5 @@ Stay updated on the latest ZetaChain news and participate in discussions on the
 
 Can’t find what you’re looking for? Feel free to ask a question on our <a href={constants.discordLink}>Discord</a>.
 Our team and community are on there and ready to help.
+
+Having problems? Check status.zetachain.com to see the latest status of the network and subscribe for automatic notifications in the future.
```

Comment:
This should be a clickable link. See the example above

Reply:
Updated to have clickable link similar to the example above